File: java-programming/examples/Prompt.java
import java.io.*;

class Prompt{
    public static void main(String[] args) throws IOException{
        System.out.print("Your name: ");
        InputStreamReader converter = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(converter);
        String name = in.readLine();
        System.out.println("Your name is: " + name);
    }
}

File: dart/src/examples/dart-intro/prompt.dart
String prompt(String text, [int count]) {
  if (count == null) {
    count = 1;
  }
  for (int i = 0; i < count; i++) {
    print(text);
  }
}

void main() {
    prompt("Your name:", 3);
    prompt("Your Cat:");
}
